src/megachatapi.h

```diff
@@ -138,6 +149,42 @@ class MegaChatSession
      */
     virtual bool hasVideo() const;
 
+    /**
+     * @brief Returns the termination code for this session
+     *
+     * Possible values are:
+     *
+     *   MegaChatCall::TERM_CODE_USER_HANGUP       = 0,    /// Normal user hangup
+     *   MegaChatCall::TERM_CODE_CALL_REQ_CANCEL   = 1,    /// Call request was canceled before call was answered
+     *   MegaChatCall::TERM_CODE_CALL_REJECT       = 2,    /// Outgoing call has been rejected by the peer OR incoming call has been rejected in
+                                            /// the current device
+     *   MegaChatCall::TERM_CODE_ANSWER_ELSE_WHERE = 3,    /// Call was answered on another device of ours
+     *   MegaChatCall::TEMR_CODE_REJECT_ELSE_WHERE = 4,    /// Call was rejected on another device of ours
+     *   MegaChatCall::TERM_CODE_ANSWER_TIMEOUT    = 5,    /// Call was not answered in a timely manner
+     *   MegaChatCall::TERM_CODE_RING_OUT_TIMEOUT  = 6,    /// We have sent a call request but no RINGING received within this timeout - no other
+                                            /// users are online
+     *   MegaChatCall::TERM_CODE_APP_TERMINATING   = 7,    /// The application is terminating
+     *   MegaChatCall::TERM_CODE_BUSY              = 9,    /// Peer is in another call
+     *   MegaChatCall::TERM_CODE_NOT_FINISHED      = 10,   /// The call is in progress, no termination code yet
+     *   MegaChatCall::TERM_CODE_DESTROY_BY_COLLISION   = 19,   /// The call has finished by a call collision
```

Comment:
Check the use-case where there is a collision at session level, in order to decide if the app needs to know about the session being destroyed by this situation.

Additionally, we should remove all the values that are not possible at session-level (only at call-level). And adjust the comment:
```suggestion
     - MegaChatCall::TERM_CODE_DESTROY_BY_COLLISION   = 19
      The session has finished by a session collision
```

Reply:
Fxied